Subject: [PATCH] Fixed kernel
---
Index: scripts/dtc/libfdt/fdt.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt.c b/scripts/dtc/libfdt/fdt.c
--- a/scripts/dtc/libfdt/fdt.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,8 +50,8 @@
  */
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/fdt_addresses.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt_addresses.c b/scripts/dtc/libfdt/fdt_addresses.c
--- a/scripts/dtc/libfdt/fdt_addresses.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt_addresses.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,8 +50,8 @@
  */
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/fdt_empty_tree.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt_empty_tree.c b/scripts/dtc/libfdt/fdt_empty_tree.c
--- a/scripts/dtc/libfdt/fdt_empty_tree.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt_empty_tree.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,8 +50,8 @@
  */
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/fdt_overlay.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt_overlay.c b/scripts/dtc/libfdt/fdt_overlay.c
--- a/scripts/dtc/libfdt/fdt_overlay.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt_overlay.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -1,7 +1,7 @@
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/fdt_ro.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt_ro.c b/scripts/dtc/libfdt/fdt_ro.c
--- a/scripts/dtc/libfdt/fdt_ro.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt_ro.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,8 +50,8 @@
  */
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/fdt_rw.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt_rw.c b/scripts/dtc/libfdt/fdt_rw.c
--- a/scripts/dtc/libfdt/fdt_rw.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt_rw.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,8 +50,8 @@
  */
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/fdt_strerror.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt_strerror.c b/scripts/dtc/libfdt/fdt_strerror.c
--- a/scripts/dtc/libfdt/fdt_strerror.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt_strerror.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,8 +50,8 @@
  */
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/fdt_sw.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt_sw.c b/scripts/dtc/libfdt/fdt_sw.c
--- a/scripts/dtc/libfdt/fdt_sw.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt_sw.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,8 +50,8 @@
  */
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/fdt_wip.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/fdt_wip.c b/scripts/dtc/libfdt/fdt_wip.c
--- a/scripts/dtc/libfdt/fdt_wip.c	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/fdt_wip.c	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,8 +50,8 @@
  */
 #include "libfdt_env.h"
 
-#include <fdt.h>
-#include <libfdt.h>
+#include "fdt.h"
+#include "libfdt.h"
 
 #include "libfdt_internal.h"
 
Index: scripts/dtc/libfdt/libfdt_internal.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/dtc/libfdt/libfdt_internal.h b/scripts/dtc/libfdt/libfdt_internal.h
--- a/scripts/dtc/libfdt/libfdt_internal.h	(revision 7b1c684552ebc0a5afa9abbdce394158ff4d7844)
+++ b/scripts/dtc/libfdt/libfdt_internal.h	(revision 84a4b8cc73966b740852b40518e6ac80b2b9e6e7)
@@ -50,7 +50,7 @@
  *     OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
  *     EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-#include <fdt.h>
+#include "fdt.h"
 
 #define FDT_ALIGN(x, a)		(((x) + (a) - 1) & ~((a) - 1))
 #define FDT_TAGALIGN(x)		(FDT_ALIGN((x), FDT_TAGSIZE))
