diff --git a/src/udev/udevd.c b/src/udev/udevd.c
--- a/src/udev/udevd.c
+++ b/src/udev/udevd.c
@@ -97,6 +97,7 @@ struct event {
         dev_t devnum;
         int ifindex;
         bool is_block;
+        bool is_firmware;
         usec_t start_usec;
         bool warned;
 };
@@ -488,6 +489,7 @@ static int event_queue_insert(struct udev_device *dev) {
         event->devpath_old = udev_device_get_devpath_old(dev);
         event->devnum = udev_device_get_devnum(dev);
         event->is_block = streq("block", udev_device_get_subsystem(dev));
+	event->is_firmware = streq("firmware", udev_device_get_subsystem(dev));
         event->ifindex = udev_device_get_ifindex(dev);
 
         log_debug("seq %llu queued, '%s' '%s'", udev_device_get_seqnum(dev),
@@ -565,7 +567,7 @@ static bool is_devpath_busy(struct event *event) {
                 }
 
                 /* parent device event found */
-                if (event->devpath[common] == '/') {
+                if (event->devpath[common] == '/' && !event->is_firmware) {
                         event->delaying_seqnum = loop_event->seqnum;
                         return true;
                 }
