diff -ur ppp-2.4.7.orig/pppd/plugins/pppol2tp/openl2tp.c ppp-2.4.7/pppd/plugins/pppol2tp/openl2tp.c
--- ppp-2.4.7.orig/pppd/plugins/pppol2tp/openl2tp.c	2017-08-14 08:41:05.356018919 +0300
+++ ppp-2.4.7/pppd/plugins/pppol2tp/openl2tp.c	2017-08-14 09:00:35.314487618 +0300
@@ -34,6 +34,7 @@
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 #include <sys/un.h>
+#include <linux/in6.h>
 #include <netinet/in.h>
 #include <signal.h>
 #include <linux/version.h>
@@ -47,6 +48,7 @@
 #include <linux/if_ether.h>
 #include <linux/ppp_defs.h>
 #include <linux/if_ppp.h>
+#include <linux/if_ppp.h>
 #include <linux/if_pppox.h>
 #include <linux/if_pppol2tp.h>
 
diff -ur ppp-2.4.7.orig/pppd/plugins/pppol2tp/pppol2tp.c ppp-2.4.7/pppd/plugins/pppol2tp/pppol2tp.c
--- ppp-2.4.7.orig/pppd/plugins/pppol2tp/pppol2tp.c	2017-08-14 08:41:05.356018919 +0300
+++ ppp-2.4.7/pppd/plugins/pppol2tp/pppol2tp.c	2017-08-14 08:58:37.894491490 +0300
@@ -34,6 +34,7 @@
 #include <net/if.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
+#include <linux/in6.h>
 #include <netinet/in.h>
 #include <signal.h>
 #include <linux/version.h>
diff -ur ppp-2.4.7.orig/pppd/plugins/rp-pppoe/plugin.c ppp-2.4.7/pppd/plugins/rp-pppoe/plugin.c
--- ppp-2.4.7.orig/pppd/plugins/rp-pppoe/plugin.c	2017-08-14 08:41:05.364018973 +0300
+++ ppp-2.4.7/pppd/plugins/rp-pppoe/plugin.c	2017-08-14 08:59:55.554491543 +0300
@@ -46,8 +46,10 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <signal.h>
+#include <linux/in6.h>
 #include <net/ethernet.h>
 #include <net/if_arp.h>
+#include <linux/if_ppp.h>
 #include <linux/ppp_defs.h>
 #include <linux/if_pppox.h>
 
