--- git/fbgrab.c	2025-02-21 15:26:17.919037817 +0300
+++ new/fbgrab.c	2025-02-21 15:36:15.333968611 +0300
@@ -175,10 +175,11 @@
         fprintf(stderr, "alpha: offset: %i, length: %i, msb_right: %i\n", fb_varinfo_p->transp.offset, fb_varinfo_p->transp.length, fb_varinfo_p->transp.msb_right);
         fprintf(stderr, "pixel format: %s\n", fb_varinfo_p->nonstd == 0 ? "standard" : "non-standard");
     }
+#if 0
     srcBlue = fb_varinfo_p->blue.offset >> 3;
     srcGreen = fb_varinfo_p->green.offset >> 3;
     srcRed = fb_varinfo_p->red.offset >> 3;
-
+#endif
     (void) close(fd);
 }
 
