diff --git a/src/v_video.cpp b/src/v_video.cpp
index 56ce654..14035b9 100644
--- a/src/v_video.cpp
+++ b/src/v_video.cpp
@@ -539,7 +539,7 @@ OVERLAY void V_DrawMemPatch(int x, int y, int scrn, const patch_t *patch,
 // static inline; other compilers have different behaviour.
 // This inline is _only_ for the function below
 #ifdef __GNUC__
-inline
+// inline
 #endif
 OVERLAY void V_DrawNumPatch(int x, int y, int scrn, int lump,
                             const byte *trans, int flags)
