
diff --git a/src/l_video_sdl.cpp b/src/l_video_sdl.cpp
index f5fb423..28278df 100644
--- a/src/l_video_sdl.cpp
+++ b/src/l_video_sdl.cpp
@@ -355,7 +355,7 @@ void I_FinishUpdate (void)
   
   // Update the display buffer
   SDL_Surface *converted =
-    SDL_ConvertSurfaceFormat(screen, SDL_PIXELFORMAT_RGBX8888, 0);
+    SDL_ConvertSurfaceFormat(screen, SDL_PIXELFORMAT_BGR565, 0);
   SDL_UpdateTexture(sdltexture, NULL, converted->pixels,
                     converted->pitch);
   SDL_FreeSurface(converted);
@@ -448,7 +448,7 @@ void I_InitGraphics(void)
                                init_flags);
   sdlrenderer = SDL_CreateRenderer(sdlwindow, -1, 0);
   SDL_RenderSetScale(sdlrenderer, multiply, multiply);
-  sdltexture = SDL_CreateTexture(sdlrenderer, SDL_PIXELFORMAT_RGBX8888,
+  sdltexture = SDL_CreateTexture(sdlrenderer, SDL_PIXELFORMAT_BGR565,
                                  SDL_TEXTUREACCESS_STREAMING, w, h);
   screen = SDL_CreateRGBSurface(0, w, h, 8, 0, 0, 0, 0);
   sdlpalette = SDL_AllocPalette(256);
